tests/unit/observers/ObjectObserver.ts

```diff
@@ -1,248 +0,0 @@
-import registerSuite = require('intern!object');
-import assert = require('intern/chai!assert');
-import has from 'src/has';
-import { Observer, PropertyEvent } from 'src/observers/interfaces';
-import { Es5Observer, Es7Observer } from 'src/observers/ObjectObserver';
-
-registerSuite({
-	name: 'observers/ObjectObserver',
-
-	'Es7Observer': {
-		'propertyChanges': function () {
-			if (!has('object-observe')) {
-				this.skip('Native Object.observe support is required for this test.');
-			}
-
-			testPropertyChanges.call(this, Es7Observer);
-		},
-
-		'when onlyReportObserved is true': function () {
-			if (!has('object-observe')) {
-				this.skip('Native Object.observe support is required for this test.');
-			}
-
-			const dfd = this.async(5000);
-			const object: { a: number; b?: number } = { a: 1 };
-
-			let isCalled: boolean = false;
-			const observer = new Es7Observer({
-				target: object,
-				listener: function (events: PropertyEvent[]): any {
-					isCalled = true;
-				}
-			});
-
-			observer.observeProperty('a');
-			object.b = 3;
-
-			setTimeout(dfd.callback(function () {
-				assert.isFalse(isCalled);
-			}), 100);
-		},
-
-		'when onlyReportObserved is false': function () {
-			if (!has('object-observe')) {
-				this.skip('Native Object.observe support is required for this test.');
-			}
-
-			const dfd = this.async(5000);
-			const object: { a: number; b?: number } = { a: 1 };
-
-			let b: number;
-			const observer = new Es7Observer({
-				onlyReportObserved: false,
-				target: object,
-				listener: function (events: PropertyEvent[]): any {
-					const target = events[0].target;
-					b = (<any> target)[events[0].name];
-				}
-			});
-
-			object.b = 3;
-
-			setTimeout(dfd.callback(function () {
-				assert.equal(b, 3);
-			}), 100);
-		},
-
-		'.removeProperty()': function () {
-			if (!has('object-observe')) {
-				this.skip('Native Object.observe support is required for this test.');
-			}
-
-			testRemoveProperty.call(this, Es7Observer);
-		},
-
-		'.destroy()': function () {
-			if (!has('object-observe')) {
-				this.skip('Native Object.observe support is required for this test.');
-			}
-
-			testDestroy.call(this, Es7Observer);
-		}
-	},
-
-	'Es5Observer': {
-		'propertyChanges': function () {
-			testPropertyChanges.call(this, Es5Observer);
-		},
-
-		'when attempting to observe an unwritable property': function () {
```

Comment:
I don't seem to see any equivalent to this test within the new test suite. Is that intentional?